src/odemis/acq/align/fastem.py

```diff
@@ -18,22 +20,254 @@
 You should have received a copy of the GNU General Public License along with
 Odemis. If not, see http://www.gnu.org/licenses/.
 """
+import logging
+import time
+from concurrent.futures import CancelledError
 
+from fastem_calibrations.configure_hw import get_config_asm, configure_asm
```

Comment:
This will fail if the fastem_calibrations module is not present, right? => Move it to the try/except too.